Shell.md

```diff
@@ -0,0 +1,32 @@
+NCI filesystems are running a Linux operating system. You will then have to work with the shell program. 
+There are several shells, we are teaching the bash shell but if you have the skills listed below in another shell program, it's fine.
+
+###Skill set #1: Navigating the filesystems
+ * Navigation:
+    * ls
+    * pwd
+    * cd
+    * . and ..
+    * command line arguments
+ * Creating, moving and deleting files and folders:
+    * rm
+    * rmdir
+    * mkdir
+    * cp
+    * mv
+    * (Note to team: Should we talk about text editors? Here or elsewhere?)
+
+###Skill set #2: Getting efficient with the shell
+ * Basic regular expressions  (mainly * ?)
```

Comment:
`*`, `?` etc. in shells are globs, not regexp

Reply:
But they are used to create regexp. at least according to this cheatsheet: https://www.rexegg.com/regex-quickstart.html 
I currently have no idea how deeply we should go into regular expressions. But there are a few things they should know about using globs.